lib/miq_automation_engine/service_models/miq_ae_service_vm.rb

```diff
@@ -1,5 +1,9 @@
 module MiqAeMethodService
   class MiqAeServiceVm < MiqAeServiceVmOrTemplate
+    def set_remote_console_url(url)
```

Comment:
Following the rubocop warning this method should be named `remote_console_url=`
`def remote_console_url=(url)`

Would also need to update the method in the VM model which would need to take two parameters.